### PR TITLE
Add ispyb recipe

### PR DESCRIPTION
**Pull request by stuartcampbell**

### CODE REVIEW

File: recipes/example/ispyb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - enum34  # [py27]
+    - pip
+    - mysql-connector-python
+    - pytest-runner
```
Comment (CJ-Wright) [inline recipes/example/ispyb/meta.yaml]: Is this a build requirement?

Comment (stuartcampbell) [reply recipes/example/ispyb/meta.yaml]: Strangely, yes it is.  Without it I can't build locally!

Comment (CJ-Wright) [reply recipes/example/ispyb/meta.yaml]: Hmm that is odd. I presume the others are in the same boat.

Comment (stuartcampbell) [reply recipes/example/ispyb/meta.yaml]: For some reason, my local env was messed up - I think I can remove `enum34` and `mysql-connector-python`

File: recipes/example/ispyb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - ispyb.connector.ws
+    - ispyb.interface
+    - ispyb.legacy
+    - ispyb.legacy.driver
+    - ispyb.legacy.driver.mysql
+    - ispyb.legacy.interface
+    - ispyb.sp
+    - ispyb.ws
+  requires:
+    - mock
```
Comment (CJ-Wright) [inline recipes/example/ispyb/meta.yaml]: Are these needed for the imports?

Comment (stuartcampbell) [reply recipes/example/ispyb/meta.yaml]: I don't think so, these were pulled in from the `skeleton` import.  Looking at the code, I think they are only needed for actually running the tests.

**Comment by CJ-Wright** [comment]
In the future please use a branch on your fork rather than pushing from master to master (I ran into an issue with this earlier today).

**Comment by CJ-Wright** [comment]
I'm reverting this merge, since the recipe is not in its own folder

**Pull request merged**